### PR TITLE
fix: revamp page

### PR DESCRIPTION
**Pull request by SputNikPlop**

### CODE REVIEW

File: web/src/Home.tsx [hunk 1]
```diff
@@ -10,33 +10,40 @@ const Home: Component = () => {
   );
   return (
     <div>
-      <div>RealtimeKit</div>
-      <div>Add a browser source in OBS: https://kit.rtirl.com/{uuid()}</div>
+      <div><h1>RealtimeKit</h1></div>
```
Comment (juaoose) [inline web/src/Home.tsx:13]: i believe these `<div>`s are redundant

**Review by juaoose — CHANGES_REQUESTED** [review]
The google play image seems to have some padding, it would be nice that both matched but I'd say that can be done separately.

File: web/src/Home.tsx (hunk 1 above)
Comment (SputNikPlop) [reply web/src/Home.tsx]: They are.

**Comment by SputNikPlop** [comment]
@juaoose yeah getting those to be align would be nice, though im unsure how to even do that

**Review by juaoose — APPROVED** [review]

**Pull request merged**